README.md

```diff
@@ -31,84 +42,58 @@ userPassword: readonly
 EOF
 ```
 
-Afterwards reload kidmd by running `systemctl reload kopano-kidmd`.
+And then passed as an additional parameter when starting `idmd` by passing `--ldif-config /etc/libregraph/idm/ldif/config.ldif`. The `config.ldif` is for service users only and the data in there is used for bind requests only, but never returned for search requests.
```

Comment:
Please make all the paths relative like `./config.ldif` to avoid people getting wrong ideas.